docs/user-guide/nginx-configuration/configmap.md

```diff
@@ -596,6 +597,12 @@ _References:_
 
 Sets the addresses on which the server will accept requests instead of *. It should be noted that these addresses must exist in the runtime environment or the controller will crash loop.
 
+## use-forwarded-headers
+
+If true, nginx passes the incoming `X-Forwarded-*` headers to upstreams. Use this option when nginx is behind another L7 proxy / load balancer that is setting these headers.
```

Comment:
(nitpick) NGINX is commonly written in capital letters.

Reply:
Done